pom.xml

```diff
@@ -14,6 +14,13 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.23</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.1.0</version>
```

Comment:
in the labs this is `<version>[5.1.0,)</version>` and the others are in a similar format. this would be preferable i think.

Reply:
cool

Reply:
example here in 2nd code block: https://github.com/Kevin-Sim/SET08103/tree/master/labs/lab08#updating-main-to-use-command-line-arguments

Reply:
Changed